frontend/public/locales/fr/translation.json

```
@@ -347,6 +347,10 @@
     "rolePresident": "Président",
     "roleTreasurer": "Trésorier",
     "roleSecretary": "Secrétaire",
+    "significantContributors": "Contributrice.eur.s important.e.s",
```

Super initiative la formule inclusive :ok_hand: 

On peut supprimer le dernier point pour faciliter la lecture, et utiliser le féminin comme suffixe pour pouvoir lire le mot d'un seul coup (contributeurice est plus facile à lire que contributriceur).

```suggestion
    "significantContributors": "Contributeur.rices important.es",
```